### ISSUE TITLE
[Bug]: `chpl-shim` produces empty json if run twice on mason

### ISSUE THREAD

**Issue by lucaferranti** [chapel-lang/chapel#25226]
### Summary of Problem

**Description:**

When running `chpl-shim mason build`  twice (without changes in the source-code in between), an empty json file is produced

### Steps to Reproduce

1. Take a mason library, preferably with source code on multiple files
2. Run

```
chpl-shim mason build
```

the json file is correctly produced and language server works

3. Run again `chpl-shim mason build` **without changing anything in the code**, this produces a new empty JSON file, which breaks the inter-file functionalities of the language server (e.g. cannot find definitions in other files)

### Configuration Information

- Output of `chpl --version`:

```
❯ chpl --version
chpl version 2.1.0 pre-release (44b3e6b437)
  built with LLVM version 14.0.0
```

- Output of `$CHPL_HOME/util/printchplenv --anonymize`:
  
 ```
CHPL_TARGET_PLATFORM: linux64
CHPL_TARGET_COMPILER: llvm
CHPL_TARGET_ARCH: x86_64
CHPL_TARGET_CPU: native
CHPL_LOCALE_MODEL: flat
CHPL_COMM: none
CHPL_TASKS: qthreads
CHPL_LAUNCHER: none
CHPL_TIMERS: generic
CHPL_UNWIND: none
CHPL_MEM: jemalloc
CHPL_ATOMICS: cstdlib
CHPL_GMP: bundled
CHPL_HWLOC: bundled
CHPL_RE2: bundled
CHPL_LLVM: system
CHPL_AUX_FILESYS: none
```
- Back-end compiler and version, e.g. `gcc --version` or `clang --version`:
```
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
Copyright (C) 2021 Free Software Foundation, Inc.
```
- (For Cray systems only) Output of `module list`:

**Comment by lydia-duncan** [comment]
@lucaferranti, just making sure you saw that Jade resolved this with their #25227.  Thanks for reporting and thanks @jabraham17 for fixing!

**Comment by lucaferranti** [comment]
Sure, I did notice the PR. Thank you for the quick fix
